utils/frame/benchmarking-cli/src/machine/hardware.rs

```diff
@@ -45,6 +45,7 @@ pub struct Requirement {
 	/// The metric to measure.
 	pub metric: Metric,
 	/// The minimal throughput that needs to be archived for this requirement.
+	#[serde(serialize_with = "serialize_throughput", deserialize_with = "deserialize_throughput")]
```

Comment:
These serialization/deserialization functions should just be moved to this file (since this is the only thing that needs those)